cirq/contrib/svg/svg.py

```diff
@@ -60,45 +62,104 @@ def _fit_horizontal(tdd: 'cirq.TextDiagramDrawer',
     return col_starts, col_widths
 
 
+def _fit_vertical(tdd: 'cirq.TextDiagramDrawer',
+                  ref_boxheight: float, row_padding: float) \
+        -> Tuple[List[float], List[float], Dict[float, int]]:
```

Comment:
Add a docstring describing the result type. Or add a module-private `NamedTuple` describing it.

Reply:
added a very informative docstring